radix-engine-interface/src/blueprints/resource/bucket.rs

```diff
@@ -142,3 +144,43 @@ impl Describe<ScryptoCustomTypeKind> for Bucket {
         well_known_scrypto_custom_types::own_bucket_type_data()
     }
 }
+
+impl Describe<ScryptoCustomTypeKind> for FungibleBucket {
+    const TYPE_ID: GlobalTypeId =
```

Comment:
These should all be well known types as they'll appear in lots of schemas, and will keep schema substates smaller :). Should be very easy to add.

I'd also be happy for the validation to be a well-known validation (to be more flexible) but that matters less if it's a well-known type.

Reply:
Desipte my temptation to argue that it's an optimisation problem and may be solved from different angels, I will go ahead and make them well known. :)